gulpfile.js

```diff
@@ -295,6 +295,9 @@ function processFile() {
 							.replace("/*INJECT SCRIPTS*/",
 								injectScripts.map((s) => `"${s}"`).join(',\n\t\t\t'))
 							.replace(/"version": "[^"]*"/, '"version": "' + argv.connectorVersion + '"');
+						if (process.env.CHROME_EXTENSION_ID && ['chrome', 'manifestv3'].includes(browser)) {
```

Comment:
Did you mean `CHROME_EXTENSION_KEY` (which you used in the other repos)?

Reply:
Well noticed